spec/inputs/gelf_spec.rb

```diff
@@ -150,7 +150,8 @@
   context "when an invalid JSON is fed to the listener" do
     subject { LogStash::Inputs::Gelf.new_event(message, "host") }
     let(:message) { "Invalid JSON message" }
-    if LogStash::Event.respond_to?(:from_java)
```

Comment:
oh boy. this is why the specs never failed on the LogStash::Event.from_json(json).each { |event| event } weirdness